libs/ui/tabs/brain/src/lib/brn-tabs-trigger.directive.ts

```diff
@@ -22,24 +22,24 @@ export class BrnTabsTriggerDirective {
 	public readonly elementRef = inject(ElementRef);
 
 	private readonly _root = inject(BrnTabsDirective);
-	private _key: string | undefined;
 
-	protected contentId: string | undefined;
-	protected labelId: string | undefined;
 	protected readonly _orientation = this._root.$orientation;
 
-	public readonly selected = computed(() => this._root.$value() === this._key);
+	triggerFor = input.required<string>({ alias: 'brnTabsTrigger' });
+	public readonly selected = computed(() => this._root.$value() === this.triggerFor());
+	protected contentId = computed(() => 'brn-tabs-content-' + this.triggerFor());
+	protected labelId = computed(() => 'brn-tabs-label-' + this.triggerFor());
 
-	@Input('brnTabsTrigger')
-	set triggerFor(key: string) {
-		this._key = key;
-		this.contentId = 'brn-tabs-content-' + this._key;
-		this.labelId = 'brn-tabs-label-' + this._key;
-		this._root.registerTrigger(key, this);
-	}
+	// TODO chaning to input signal makes it not compatible with FocusKeyManager as it expects `disabled: boolean`
```

Comment:
@marcjulian what do you mean by this comment? Does this break focus management?

Reply:
Changing the @input `disabled` to `disabled = input(false);` causes a type error with the `FocusKeyManager` in `BrnTabsListDirective`. It won't compile otherwise, thats why I left it as @input

![CleanShot 2024-02-21 at 18 02 14](https://github.com/goetzrobin/spartan/assets/8985933/ecff5ae7-e429-4a25-8f3b-f2a27374cbc9)

Reply:
Oh I see that makes sense!